src/coreclr/vm/stubmgr.cpp

```diff
@@ -1563,13 +1568,7 @@ BOOL RangeSectionStubManager::DoTraceStub(PCODE stubStartAddress, TraceDestinati
 #ifdef DACCESS_COMPILE
         DacNotImpl();
 #else
-#if defined(TARGET_ARM64) && defined(__APPLE__)
-        // On ARM64 Mac, we cannot put a breakpoint inside of ExternalMethodFixupPatchLabel
```

Comment:
Delete all occurrences of `ExternalMethodFixupPatchLabel`?

Reply:
Are you referring to all of the ExtermalMethodFixupPatchLabel in each one of the assembly racks?